extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/supervisor/KafkaSupervisor.java

```diff
@@ -255,6 +257,12 @@ public boolean apply(TaskRunnerWorkItem taskRunnerWorkItem)
       }
     };
 
+    this.futureTimeout = Math.max(
+        120,
```

Comment:
Might as well make this a constant too, along with the other stuff you made constants.